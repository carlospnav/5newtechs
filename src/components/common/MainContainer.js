import styled from "styled-components"
import { flexGroup } from "theme/utils"

export default styled.div`
  ${flexGroup("column")};
  padding: 0 30px;
  width: 100%;
`
