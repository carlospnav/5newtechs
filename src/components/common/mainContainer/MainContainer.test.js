import { shallow } from "enzyme"
import MainContainer from "./MainContainer"
import "jest-styled-components"

describe("<MainContainer>", () => {
  it("should render a detailed MainContainer properly.", () => {
    const wrapper = shallow(<MainContainer details/>)
    expect(wrapper).toMatchSnapshot()
  })

  it("should render a regular MainContainer properly.", () => {
    const wrapper = shallow(<MainContainer />)
    expect(wrapper).toMatchSnapshot()
  })
})