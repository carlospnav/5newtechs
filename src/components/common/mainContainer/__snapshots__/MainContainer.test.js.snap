// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainContainer> should render a detailed MainContainer properly. 1`] = `
.c0 {
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  max-width: 1120px;
  margin: 80px auto;
  z-index: 0;
}

@media (min-width:600) {
  .c0 {
    padding: 0 30px;
  }
}

<main
  className="c0"
/>
`;

exports[`<MainContainer> should render a regular MainContainer properly. 1`] = `
.c0 {
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: null;
  -webkit-justify-content: null;
  -ms-flex-pack: null;
  justify-content: null;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  width: 100%;
  -webkit-flex-wrap: nowrap;
  -ms-flex-wrap: nowrap;
  flex-wrap: nowrap;
  max-width: 1120px;
  margin: 0 auto;
  z-index: 0;
}

<main
  className="c0"
/>
`;
